tx_sign.py

```diff
@@ -1,3 +1,5 @@
+"""Prepare transaction on a network connected machine."""
```

Comment:
Would you update this header before I merge ? Anyway thanks for the PR, appreciated !